src/decorators/index.js

```diff
@@ -85,9 +85,13 @@ function keydownScoped( ...args ) {
  * @return {Function} The decorated class or method
  */
 function keydown( ...args ) {
-  return _decorator( methodWrapper, ...args );
+  return _decorator( methodWrapper, {}, ...args );
+}
+
+function keydownGlobal( ...args ) {
+  return _decorator( methodWrapper, { global: true }, ...args);
```

Comment:
This will clash a bit when we start taking `options` from the decorator. `options` will then be last arg.